db/migrate/20180102134015_add_external_identifier_to_person.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class AddExternalIdentifierToPerson < ActiveRecord::Migration[5.0]
+  def change
+    add_column :people, :external_identifier, :string
+
+    Person.all.each do |person|
+      next if person.external_identifier
```

Comment:
Volgens mij, maar dat zou je even moeten checken, roep je de `after_initialize` van Person aan op het moment dat je de variabele `person` hier instantieert. Dus dan kom je vvolgens mij nooit bij de onderstaande regels (wat wel chill zou zijn want die code staat ook in het model zeg maar).

Het kan ook zijn dat je dingen die je in de after_initialize aanpast niet direct opslaat naar de database en hier dus nog wel person.save! moet aanroepen ondanks dat het lijkt alsof je niks hebt aangepast. Dat zou je even moeten uitzoeken.

Reply:
nice, ja dat is wel mooi. Hij slaat idd die dingen niet direct op.